creation/web_base/factoryEntryStatusNow.html

```diff
@@ -884,19 +884,19 @@
             ) {
               document
                 .getElementsByTagName("div")
-                [
-                  index
-                ].removeChild(document.getElementsByTagName("div")[index].firstChild);
+                [index].removeChild(
+                  document.getElementsByTagName("div")[index].firstChild,
```

Comment:
pre-commit did it. I did not touch this file. After removing the changes many times I decided to include them in this commit since I was also updating pre-commit